beetsplug/lastgenre/genres-tree.yaml

```diff
@@ -719,6 +735,7 @@
             - street punk
             - thrashcore
         - horror punk
+        - oi!
```

Comment:
Just checking: is this really a genre?

Reply:
See here: https://en.wikipedia.org/wiki/Oi!

Reply:
Awesome and fascinating!